types/react-virtualized/tsconfig.json

```diff
@@ -5,6 +5,11 @@
             "es6",
             "dom"
         ],
+        "paths": {
+            "react": [
+                "react/v17"
+            ]
+        },
```

Comment:
https://github.com/bvaughn/react-virtualized/blob/2e962d8f8aebc22cb1168a8d147bcaa1d27aa326/package.json#L150

Reply:
They have added 18 now:
https://github.com/bvaughn/react-virtualized/releases/tag/v9.22.4

Does it mean that this entry can be removed?